src/Console/CommandRunner.php

```diff
@@ -319,6 +319,10 @@ protected function resolveName(CommandCollection $commands, ConsoleIo $io, ?stri
     protected function runCommand(CommandInterface $command, array $argv, ConsoleIo $io): ?int
     {
         try {
+            $eventManager = $this->getEventManager();
+            $eventManager = $this->app->consoleEvents($eventManager);
+            $eventManager = $this->app->pluginConsoleEvents($eventManager);
```

Comment:
```suggestion
            $eventManager = $this->app->events($eventManager);
            $eventManager = $this->app->pluginEvents($eventManager);
```

We could call `events` hook here as well. If applications need events bound only in commands/http they can use `PHP_SAPI`.

Reply:
I see, so the same like its done with bootstrap